src/main/java/clojure/osgi/RunnableWithException.java

```diff
@@ -11,6 +11,6 @@
 
 package clojure.osgi;
 
-public interface RunnableWithException {
-	Object run() throws Exception;
+public interface RunnableWithException<T> {
+	T run() throws Exception;
```

Comment:
This is the change of primary substance. With the return type of runnable blocks parameterized, we benefit from elimination of type casting and compile-time type checking.

Reply:
... we benefit from human-error-prone type casting being replaced with compile-time type checking.